testing/adios2/bindings/C/TestBPWriteTypes.cpp

```diff
@@ -8,20 +8,32 @@
  *      Author: wgodoy
  */
 
+#ifdef ADIOS2_HAVE_MPI
```

Comment:
The `#ifdef` should be after including `adios2_c.h` since none of the `ADIOS2Config.h` that actually defines the options isn't included until then.

Reply:
Done.